independent-publisher-connectors/MS Graph Groups and Users/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1151 @@
+                        "description": "Member's email",
+                        "title": "Mail"
+                      },
+                      "givenName": {
+                        "type": "string",
+                        "description": "Member's first name",
+                        "title": "Given Name"
+                      },
+                      "surname": {
+                        "type": "string",
+                        "description": "Member's last name",
+                        "title": "Surname"
+                      },
+                      "businessPhones": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        },
+                        "description": "businessPhones"
+                      },
+                      "mobilePhone": {
+                        "type": "string",
+                        "description": "mobilePhone",
+                        "title": "Mobile Phone"
+                      },
+                      "officeLocation": {
+                        "type": "string",
+                        "description": "officeLocation",
+                        "title": "Office Location"
+                      },
+                      "preferredLanguage": {
+                        "type": "string",
+                        "description": "Preferred language as configured in Azure AD.",
+                        "title": "Preferred Language"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            },
+            "headers": {
+              "\"cache-control\"": {
+                "description": "\"cache-control\"",
+                "type": "string"
+              },
+              "\"client-request-id\"": {
+                "description": "\"client-request-id\"",
+                "type": "string"
+              },
+              "\"content-type\"": {
+                "description": "\"content-type\"",
+                "type": "string"
+              },
+              "\"request-id\"": {
+                "description": "\"request-id\"",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "List Direct Group Members",
+        "operationId": "ListDirectGroupMembers",
+        "description": "Retrieve direct members of a group with count",
+        "parameters": [
+          {
+            "name": "group-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Object ID of the AAD group",
+            "x-ms-summary": "Object ID of the AAD group",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "jobTitle ne null",
```

Comment:
@jagjani03 can you please check if this text is correct? Shouldn't it be `jobTitle is not null`?

Reply:
@v-EgorKozhadei This is correct. It's oData syntax. Thanks!